sandbox/install-client-archives.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+# To run this script, you must first set the version number.
+# For more information, see README.md
+
+if [[ -z "${VERSION}" ]]; then
+    echo "You must set the release version number. For example:"
```

Comment:
Should we add the URL to the release pages here, to make it easy for folks to look up which version number they're interested in?

Reply:
Good idea, thanks. I'll add the URL.